cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapperWrapper.java

```diff
@@ -126,8 +136,7 @@ public void run(Context context) throws IOException, InterruptedException {
     try {
       basicMapReduceContext.flushOperations();
     } catch (Exception e) {
-      LOG.error("Failed to flush operations at the end of mapper of {}", basicMapReduceContext, e);
-      throw Throwables.propagate(e);
+      throw new IOException("Failed to flush operations at the end of mapper of " + basicMapReduceContext, e);
```

Comment:
This is sent out as a part of the exception. Context contains name, jobid and spec. This was a log which is changed to exception here. This is not part of the user log.